### DISCUSSION TITLE
All applications and profiles missing after updating to V1.8.7

### DISCUSSION THREAD

**Discussion by rb2k5** [Codectory/AutoActions#80] [Q&A]
I've just updated AutoHDR to V1.8.7 only to find all the apps and profiles that I had added are no longer there.
I have a "Backup_Settings_20211228093337.xml.bak" file in the AutoHDR directory. 
Can I use that to restore all the settings?

**Comment by Codectory** [comment]
Which was your previous version? Please send me the backup file.

**Reply by rb2k5** [reply]
I think it was 1.8.4
[Backup_Settings_20211228093337.xml.bak.txt](https://github.com/Codectory/AutoHDR/files/7784064/Backup_Settings_20211228093337.xml.bak.txt)

**Comment by Codectory** [comment]
Did you remove any games since 1.8.4?

**Comment by rb2k5** [comment]
Dont think so, but I'll check to see if any of the apps are no longer there.
Where is the settings (xml) file normally kept?

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Reply by Codectory** [reply]
Rename the backup file to UserSettings.json and try this one:
[Release_AutoHDR_1.8.8_x64.zip](https://github.com/Codectory/AutoHDR/files/7784152/Release_AutoHDR_1.8.8_x64.zip)
[Release_AutoHDR_1.8.8_x86.zip](https://github.com/Codectory/AutoHDR/files/7784154/Release_AutoHDR_1.8.8_x86.zip)

**Reply by rb2k5** [reply]
That did it! 
Thanks.
Looks like you were right about an app being removed. Got a [PackageNotFound]Tetris Effect: Connected entry at the bottom.
I removed it and re-added

**Reply by rb2k5** [reply]
Looks like it might have been a Windows Store app update. In the .bak file it had -
        "UWPFullPackageName": "48710EnhanceIncorporated.TRIP2.0_1.2.70.0_x64__63vy8jfbpt4dt",
Since re=adding the app its now -
        "UWPFullPackageName": "48710EnhanceIncorporated.TRIP2.0_1.2.71.0_x64__63vy8jfbpt4dt",

**Reply by Codectory** [reply]
Since 1.8.7, there is a fallback method. If the FullPackageName cannot be found, AutoHDR will look for FamilyPackageName and ApplicationID, which is a little bit slower but it will do the job.

**Reply by rb2k5** [reply]
Thanks again for the really quick response. 
Much appreciated.

**Reply by rb2k5** [reply]
Hi, Just to add I had the same issue this morning updating from 1.8.8 to 1.9.1
Same fix though - renamed Backup_Settings_20220102115945.xml.bak to UserSettings.json and all was well.
